tests/lib/rules/no-replace-test-comments.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview No &#39;Replace this with your real tests&#39; comments
+ * @author Jay Gruber
+ */
```

Comment:
Could you remove this whole comment block from the top of the files? We don't use it in our files.

Reply:
👍 Yeoman spit that out from [generator-eslint](https://github.com/eslint/generator-eslint)
🔥 blazing fast feedback, thanks!